### PR TITLE
Updated documentation (Once more with feeling)

### PR DESCRIPTION
**Pull request by craigatgoogle**

### CODE REVIEW

File: docs/KubernetesInstallation.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+enables users to install Jenkins into a Kubernetes cluster using
+[Helm](https://github.com/helm/helm). Included in the chart's configuration
+values is the option to specify a set of Jenkins plugins to be installed by
+default. This section provides guidance on utilizing this chart to install the
+GKE plugin.
+
+*   If installing into a cluster running within GKE, we suggest following Google
+    Cloud Solutions'
+    [tutorial](https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial)
+    for setting up Jenkins on GKE. **Note**: The chart version specified in the
```
Comment (stephenashank) [inline docs/KubernetesInstallation.md]: Thanks for this

Comment (craigatgoogle) [reply docs/KubernetesInstallation.md]: np :)

File: docs/KubernetesInstallation.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    tutorial is out of date, we suggest users find a version which best suites
+    their use-case.
+
+*   Modify the `values.yaml` to be used when running `helm install --values` to
+    include the following configuration values:
+    ```yaml 
+        Agent:
+            Image: gcr.io/jenkins-gke-plugin/jenkinsagent ImageTag: latest
```
Comment (stephenashank) [inline docs/KubernetesInstallation.md]: Move ImageTag to next line

File: docs/KubernetesInstallation.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+!--
```
Comment (stephenashank) [inline docs/KubernetesInstallation.md]: Missing an opening `<` here

File: docs/KubernetesInstallation.md (hunk 2 above)
Comment (craigatgoogle) [reply docs/KubernetesInstallation.md]: Good catch, thx.

File: docs/KubernetesInstallation.md (hunk 3 above)
Comment (craigatgoogle) [reply docs/KubernetesInstallation.md]: Fixed.

**Pull request merged**